src/containers/add-attachments/AddAttachments.tsx

```diff
@@ -61,6 +69,29 @@ const AddAttachments: FC = () => {
     setInputValue('')
   }
 
+  const createAttachments = useCallback(
```

Comment:
in service you have "createAttachment" not "createAttachments", so rename it

Reply:
Fixed